pro/src/new_components/OfferIndividualForm/UsefulInformations/__specs__/UsefulInformations.spec.tsx

```diff
@@ -93,4 +144,99 @@ describe('OfferIndividual section: UsefulInformations', () => {
     await screen.findByRole('heading', { name: 'Informations pratiques' })
     expect(screen.getByLabelText('Rayonnement national')).toBeInTheDocument()
   })
+
+  it('should contain withdrawal ticket informations when subcategory is an event', async () => {
+    // for an event subCategory
+    initialValues.subcategoryId = 'A-A'
+
+    renderUsefulInformations({
+      initialValues,
+      onSubmit,
+      props,
+    })
+
+    await screen.findByRole('heading', { name: 'Informations pratiques' })
+    expect(
+      await screen.findByText(
+        'Comment les billets, places seront-ils transmis ?'
+      )
+    ).toBeInTheDocument()
+
+    // should contain sent date information when tickets are sent by mail
+    userEvent.click(await screen.findByText('Envoi par e-mail'))
+    expect(await screen.findByText('Date d’envoi')).toBeInTheDocument()
+
+    // should contain withdrawal hour information when tickets are to withdraw on place
+    userEvent.click(
+      await screen.findByText('Retrait sur place (guichet, comptoir ...)')
+    )
+    expect(await screen.findByText('Heure de retrait')).toBeInTheDocument()
+  })
+
+  it('should display an error when the user has not filled all required fields', async () => {
+    // for an event subCategory
+    initialValues.subcategoryId = 'A-A'
+
+    renderUsefulInformations({
+      initialValues,
+      onSubmit,
+      props,
+    })
+
+    expect(
+      screen.queryByText('Vous devez cocher l’une des options ci-dessus')
+    ).not.toBeInTheDocument()
+    expect(
+      screen.queryByText('Vous devez choisir une date d’envoi')
+    ).not.toBeInTheDocument()
+    expect(
+      screen.queryByText('Vous devez choisir une heure de retrait')
+    ).not.toBeInTheDocument()
+
+    await userEvent.tab()
+
+    waitFor(() => {
```

Comment:
```suggestion
    await waitFor(() => {
```
Sinon jest n'attend pas la fin de l’exécution du `waitFor` pour valider le test.
Ici on voit que l'ajout du await casse le test, normal car le tab d'avant n'a pas lancer la validation du formulaire formik.